src/store/Business/getters.ts

```diff
@@ -0,0 +1,160 @@
+import { CorpTypeCd, EntityState } from '@/enums'
+import { BusinessStateIF, BusinessWarningIF } from '@/interfaces'
+
+export default {
+  /** The business number (aka Tax ID). */
+  getBusinessNumber (state: BusinessStateIF): string {
+    return state.businessNumber
+  },
+
+  /** The business warnings list. */
+  getBusinessWarnings (state: BusinessStateIF): Array<BusinessWarningIF> {
+    return state.businessWarnings
+  },
+
+  /** The entity founding date. */
+  getEntityFoundingDate (state: BusinessStateIF): Date {
+    return state.entityFoundingDate
+  },
+
+  /** The entity name. */
+  getEntityName (state: BusinessStateIF): string {
+    return state.entityName
+  },
+
+  /** The entity state. */
+  getEntityState (state: BusinessStateIF): EntityState {
+    return state.entityState
+  },
+
+  /** The entity type. */
+  getEntityType (state: BusinessStateIF): CorpTypeCd {
+    return state.entityType
+  },
+
+  /** The business identifier (aka Incorporation Number). */
+  getIdentifier (state: BusinessStateIF): string {
+    return state.identifier
+  },
+
+  /** The last address change date. */
+  getLastAddressChangeDate (state: BusinessStateIF): string {
+    return state.lastAddressChangeDate
+  },
+
+  /** The last annual report change date. */
+  getLastAnnualReportDate (state: BusinessStateIF): string {
+    return state.lastAnnualReportDate
+  },
+
+  /** The last director change date. */
+  getLastDirectorChangeDate (state: BusinessStateIF): string {
+    return state.lastDirectorChangeDate
+  },
+
+  /** The state filing URL (may be null). */
+  getStateFilingUrl (state: BusinessStateIF): string {
+    return state.businessInfo.stateFiling
+  },
+
+  /** Is true of the business has a court order filing */
+  hasCourtOrders (state: BusinessStateIF): boolean {
+    return state.hasCourtOrders
+  },
+
+  /** Is True if a firm has at least one "compliance" warning. */
+  hasComplianceWarning (_state: BusinessStateIF, getters): boolean {
```

Comment:
Using `_state` so this unused parameter doesn't cause a warning.

Note the use of additional parameter `getters` (from this module). There's also `rootState` and `rootGetters` (from rest of store).